libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -962,23 +964,22 @@ - (SFSDKUserAgentCreationBlock)defaultUserAgentString {
                                  (qualifier != nil ? qualifier : @""),
                                  uid,
                                  [[[SFSDKAppFeatureMarkers appFeatures].allObjects sortedArrayUsingSelector:@selector(caseInsensitiveCompare:)] componentsJoinedByString:@"."],
-                                 webViewUserAgent
+                                 self.webViewUserAgent == nil ? @"" : self.webViewUserAgent
                                  ];
         return myUserAgent;
     };
 }
 
-- (NSString *)getUIWebViewUserAgent {
-    static NSString *webViewUserAgent = nil;
+- (void)computeWebViewUserAgent {
```

Comment:
@wmathurin Does the actual base agent String change drastically or is it the same?

Reply:
The web view user agent is appended at the end of the Mobile SDK generated user agent. Are you asking if UI web view and WK web view produce different user agent?